core/iwasm/interpreter/wasm_interp_fast.c

```diff
@@ -3695,8 +3698,10 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
                 frame->operand + cur_wasm_func->const_cell_num;
 
             /* Initialize the consts */
-            word_copy(frame->operand, (uint32 *)cur_wasm_func->consts,
-                      cur_wasm_func->const_cell_num);
+            if (cur_wasm_func->const_cell_num > 0) {
```

Comment:
Should also check the num parameter in line 3581

Reply:
oops, i somehow misread that it couldn't be 0.
fixed.